scripts/ipc/mine-subnet.sh

```diff
@@ -12,4 +12,4 @@ fi
 IMG=$1
 
 echo "[*] Starting to mine on validator"
-docker exec -it $IMG eudico mir validator run --membership=onchain --nosync --ipcagent-url=http://host.docker.internal:3030/json_rpc
+docker exec -e MIR_INTERCEPTOR_OUTPUT="mir-event-logs" -it $IMG eudico mir validator run --membership=onchain --nosync --ipcagent-url=http://host.docker.internal:3030/json_rpc
```

Comment:
Do we really want to have the interceptor enabled always? Won't this increase the size requirement a lot?

Reply:
Fixed

Reply:
That was me; I thought the branch was just for our tests and didn't realise we would merge the thing into dev :)